src/main/kotlin/uk/co/ben_gibson/git/link/platform/Platform.kt

```diff
@@ -48,7 +49,9 @@ class BitbucketCloud() : Platform(
 class BitbucketServer() : Platform(
     UUID.fromString("dba5941d-821c-49b3-83b0-75deb9462acb"),
     message("platform.bitbucket.server.name"),
-    Icons.BITBUCKET
+    Icons.BITBUCKET,
+    setOf(),
+    Pattern.compile(".*bitbucket.*", Pattern.CASE_INSENSITIVE)
```

Comment:
Does this have any impact on the `BitbucketCloud` detection? Or does that get checked first and match `bitbucket.org` before reaching this check

Reply:
No impact.
Domain patterns are checked in a loop coming after domain loop.